internal/blocksync/reactor.go

```diff
@@ -517,8 +517,16 @@ func (r *Reactor) poolRoutine(ctx context.Context, stateSynced bool, blockSyncCh
 			// NOTE: We can probably make this more efficient, but note that calling
 			// first.Hash() doesn't verify the tx contents, so MakePartSet() is
 			// currently necessary.
-			if err = state.Validators.VerifyCommitLight(chainID, firstID, first.Height, second.LastCommit); err != nil {
-				err = fmt.Errorf("invalid last commit: %w", err)
```

Comment:
Not sure I follow you.
In the old code, line 521 is always overwriting the error coming from line 520 (because of the condition in the "if").
In fairness, I don't even get the reason for 521 to exist in the first place, do you? If not, it's usually a good idea to leave it as it was.

Reply:
I suppose it was just to add more context to the error message. I don't think it's really necessary